src/pykeen/training/training_loop.py

```diff
@@ -1024,6 +1104,18 @@ def _load_state(self, path: Union[str, pathlib.Path]) -> None:
                 "be deterministic.",
             )
 
+        # If the checkpoint was saved with a best epoch model from the early stopper, this model has to be retrieved
+        best_epoch_model_file_path = None
+        best_epoch = None
+        if checkpoint['best_epoch_model_checkpoint'] is not None:
```

Comment:
maybe better to use .get() in case it's not there?

Reply:
since this is only used internally and it's assumed it will be there, it's fine to leave as is as well